pulp_container/tests/functional/utils.py

```diff
@@ -157,18 +157,16 @@ def gen_container_remote(url=REGISTRY_V2_FEED_URL, **kwargs):
     return gen_remote(url, upstream_name=kwargs.pop("upstream_name", REPO_UPSTREAM_NAME), **kwargs)
 
 
-def get_docker_hub_remote_blobsums(upstream_name=REPO_UPSTREAM_NAME):
+def get_blobsums_from_remote_registry(upstream_name=REPO_UPSTREAM_NAME):
     """Get remote blobsum list from dockerhub registry."""
-    token_url = (
-        "https://auth.docker.io/token"
-        "?service=registry.docker.io"
-        "&scope=repository:library/{0}:pull"
-    ).format(upstream_name)
+    token_url = "https://ghcr.io/token?service=ghcr.io&scope=repository:{0}:pull".format(
+        upstream_name
+    )
     token_response = requests.get(token_url)
     token_response.raise_for_status()
     token = token_response.json()["token"]
 
-    blob_url = ("{0}/v2/library/{1}/manifests/latest").format(REGISTRY_V2_FEED_URL, upstream_name)
+    blob_url = ("{0}/v2/{1}/manifests/latest").format(REGISTRY_V2_FEED_URL, upstream_name)
     response = requests.get(blob_url, headers={"Authorization": "Bearer " + token})
```

Comment:
add aceept header for manifest.2 mediatype and the tests should pass

Reply:
The GH Packages Registry does not consider `Accept` headers and returns the content like it was uploaded.